packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics-latencies.ts

```diff
@@ -267,6 +267,17 @@ export default class CallDiagnosticLatencies {
     return undefined;
   }
 
+  /**
+   * Interstitial To Media OK JMT
+   * @returns - latency
+   */
+  public getInterstitialToMediaOKJMT() {
+    return this.getDiffBetweenTimestamps(
+      'internal.client.interstitial-window.click.joinbutton',
```

Comment:
just a thought: who is going to call the metrics with the 'internal.client.interstitial-window.click.joinbutton' event, it's going to be the web app, right? What about other apps using the JS-SDK. When they join meetings we will be sending metrics for Call Analyzer, but there is no guarantee that the other apps will send any of the internal events. How is it going to work with these other apps? Is Call Analyzer OK with SDK just sending undefined values for the affected latencies?